vnext/Shared/Hasher.cpp

```diff
@@ -0,0 +1,78 @@
+#include "pch.h"
+#include "Hasher.h"
+
+#ifndef NT_SUCCESS
+#define NT_SUCCESS(Status) ((NTSTATUS)(Status) >= 0)
+#endif // NT_SUCCESS
+
+namespace Microsoft::ReactNative {
+
+  SHA256Hasher::SHA256Hasher()
+  {
+    NTSTATUS status;
+    if (!NT_SUCCESS(status = BCryptOpenAlgorithmProvider(&m_hAlg, BCRYPT_SHA256_ALGORITHM, NULL, 0)))
```

Comment:
> if (!NT_SUCCESS [](http://example.com/codeflow?start=3&length=16)

It is a repetitive pattern. I wonder if it is worth adding a macro that takes care of it.